docs/_posts/akrztrk/2024-03-04-sbiobertresolve_snomed_bodyStructure_en.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: model
+title: Sentence Entity Resolver for SNOMED (sbiobertresolve_snomed_bodyStructure)
+author: John Snow Labs
+name: sbiobertresolve_snomed_bodyStructure
+date: 2024-03-04
+tags: [licensed, en, resolver, snomed, bodystructure]
+task: Entity Resolution
+language: en
+edition: Healthcare NLP 5.3.0
+spark_version: 3.0
+supported: true
+annotator: SentenceEntityResolverModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps extracted medical (anatomical structure) entities to SNOMED codes (body structure version) using `sbiobert_base_cased_mli` BERT sentence embeddings
+
+## Predicted Entities
+
+
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/sbiobertresolve_snomed_bodyStructure_en_5.3.0_3.0_1709543980434.zip){:.button.button-orange.button-orange-trans.arr.button-icon.hidden}
+[Copy S3 URI](s3://auxdata.johnsnowlabs.com/clinical/models/sbiobertresolve_snomed_bodyStructure_en_5.3.0_3.0_1709543980434.zip){:.button.button-orange.button-orange-trans.button-icon.button-copy-s3}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+  
+```python
+document_assembler = DocumentAssembler()\
+    .setInputCol("text")\
+    .setOutputCol("document")
+
+sentence_detector = SentenceDetectorDLModel.pretrained("sentence_detector_dl_healthcare", "en", "clinical/models")\
+    .setInputCols(["document"])\
+    .setOutputCol("sentence")
+
+tokenizer = Tokenizer()\
+    .setInputCols(["sentence"])\
+    .setOutputCol("token")\
+
+word_embeddings = WordEmbeddingsModel.pretrained("embeddings_clinical", "en", "clinical/models")\
+    .setInputCols(["sentence", "token"])\
+    .setOutputCol("embeddings")
+
+ner_jsl = MedicalNerModel.pretrained("ner_jsl", "en", "clinical/models") \
+    .setInputCols(["sentence", "token", "embeddings"]) \
+    .setOutputCol("ner_jsl")
+
+ner_jsl_converter = NerConverterInternal() \
+    .setInputCols(["sentence", "token", "ner_jsl"]) \
+    .setOutputCol("ner_jsl_chunk")\
+    .setWhiteList(["External_body_part_or_region"])\
```

Comment:
why didn't you add Internal_organ_or_component?

Reply:
Added